edx_django_utils/monitoring/transactions.py

```diff
@@ -0,0 +1,94 @@
+"""
+This is a collection of transaction-related monitoring utilities.
+
+Usage:
+
+    from edx_django_utils import monitoring as monitoring_utils
+    ...
+    monitoring_utils.ignore_transaction()
```

Comment:
Nit: I think I'd go with simpler import, and I added a comment:
```
from edx_django_utils.monitoring import ignore_transaction
...
# called from inside a view you want ignored
ignore_transaction()
```

Reply:
Fixed :+1: